test/api/dependency/api_test.go

```diff
@@ -15,6 +15,8 @@ func TestDependencyCRUD(t *testing.T) {
 			assert.Must(t, Application.Create(&sample.ApplicationFrom))
 			assert.Must(t, Application.Create(&sample.ApplicationTo))
 
+			// Create a check for reverse dependency
```

Comment:
Looks to be a not needed comment here.

Reply:
Oops forgot to delete it. Was trying to add the test in the CRUD test before.